public/approval/v1.0/openapi.json

```diff
@@ -1082,6 +1077,16 @@
                         "type": "string",
                         "description": "Comments for action",
                         "readOnly": true
+                    },
+                    "created_at": {
+                        "type": "string",
+                        "format": "date-time",
+                        "description": "Timestamp of creation"
+                    },
+                    "updated_at": {
```

Comment:
Are all these timestamps required by the serializer or do you add them just because you think good to have?

Reply:
A few of them were required by the unit tests, but I could also change those tests to look at different fields as well. I'm open to either option. On the Catalog side we added created_at/updated_at to most responses it looked like, I'll double check to see if it's on every response for sure.

Reply:
@bzwei on catalog we ended up exposing timestamps for most of the model backed endpoints. The thought was that dates were useful in general as part of a default json payload. 

We also were expecting a number of timestamps in the original specs, so that also contributed to the changes in the `openapi.json` spec file.